src/NServiceBus.Msmq.AcceptanceTests/NServiceBus.Msmq.AcceptanceTests.csproj

```diff
@@ -89,6 +89,9 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
@danielmarbach blech, I press "Add service reference" or some crap like that by mistake... sigh. Let me see if I can revert it